text/0000-rust-governance.md

```diff
@@ -0,0 +1,719 @@
+
+* Setting policy on what changes in the subteam area require RFCs, and reviewing
+  direct PRs for changes that do not require an RFC.
+
+* Delegating *reviewer rights* for the subteam area. The ability to `r+` is not
+  limited to team members, and in fact earning `r+` rights is a good stepping
+  stone toward team membership. Each team should set reviewing policy, manage
+  reviewing rights, and ensure that reviews take place in a timely manner.
+  (Thanks to Nick Cameron for this suggestion.)
+
+Subteams make it possible to involve a larger, more diverse group in the
+decision-making process. In particular, **they should involve a mix of**:
+
+* Rust project leadership, in the form of at least one core team member (the
+  leader of the subteam).
+
+* Area experts: people who have a lot of interest and expertise in the subteam
+  area, but who may be far less engaged with other areas of the project.
+
+* Stakeholders: people who are strongly affected by decisions in the
+  subteam area, but who may not be experts in the design or
+  implementation of that area. *It is crucial that some people heavily
+  using Rust for applications/libraries have a seat at the table, to
+  make sure we are actually addressing real-world needs.*
+
+Members should have demonstrated a good sense for design and dealing with
+tradeoffs, an ability to work within a framework of consensus, and of course
+sufficient knowledge about or experience with the subteam area. Leaders should
+in addition have demonstrated exceptional communication, design, and people
+skills. They must be able to work with a diverse group of people and help lead
+it toward consensus and execution.
+
+Each subteam is led by a member of the core team. **The leader is responsible for**:
+
+* Setting up the subteam:
+
+    * Deciding on the initial membership of the subteam (in consultation with
+      the core team).
+
+    * Working with subteam members to determine and publish subteam policies and
+      mechanics.
+
+* Communicating core team vision downward to the subteam.
+
+* Alerting the core team to subteam RFCs that need global, cross-cutting
+  attention, and to RFCs that have entered the "final comment period" (see below).
+
+* Ensuring that RFCs and PRs are progressing at a reasonable rate, re-assigning
+  shepherds/reviewers as needed.
+
+* Making final decisions in cases of contentious RFCs that are unable to reach
+  consensus otherwise (should be rare).
+
+The way that subteams communicate internally and externally is left to each
+subteam to decide, but:
+
+* Technical discussion should take place as much as possible on public forums,
+  ideally on RFC/PR threads and tagged discuss posts.
+
+* Each subteam will have a dedicated discuss forum tag.
+
+* Subteams should have some kind of regular meeting or other way of making
+  decisions.
+
+* Subteams should regularly publish the status of RFCs, PRs, and other news
+  related to their area.
+
```

Comment:
I think an explicit recommendation for subteams to involve people outside of the subteam in discussions/decisions whenever possible would be nice to have here. (It's probably going to happen in practice anyway)

The subteams should be viewed as place where the discussion focuses, not an authority figure that swoops in and hands out decisions.